add.go

```diff
@@ -60,9 +62,10 @@ func (a *AddRequest) Attribute(attrType string, attrVals []string) {
 }
 
 // NewAddRequest returns an AddRequest for the given DN, with no attributes
-func NewAddRequest(dn string) *AddRequest {
+func NewAddRequest(dn string, controls []Control) *AddRequest {
```

Comment:
Then it would be different to the existing [NewDelRequest](https://github.com/go-ldap/ldap/blob/master/del.go#L30). I'd rather provide a "migrating v2->v3" doc

Reply:
Ok